apps/desktop-wallet/src/storage/store.ts

```diff
@@ -45,7 +45,14 @@ export const store = configureStore({
     [modalSlice.name]: modalSlice.reducer
   },
   middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware().concat(settingsListenerMiddleware.middleware).concat(networkListenerMiddleware.middleware)
+    getDefaultMiddleware({
+      serializableCheck: {
```

Comment:
We could maybe add a little comment here, to explain that this is because we need to use functions as props.

Reply:
ok!

Reply:
Done!